### PR TITLE
Framework working with Carthage and CocoaPods.

### PR DESCRIPTION
**Pull request by BogdanTC**
Framework functional with dependency managers(CocoaPods/Carthage), demo app removed from branch.

### CODE REVIEW

File: AutoCompletion.podspec [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+s.version      = "1.2.1"
+s.summary      = "Text auto completion framework."
+s.description  = "Library can be used via cocoapods in any project."
+s.homepage     = "https://github.com/3pillarlabs/ios-autocomplete/tree/AutoCompleteFramework"
+s.license      = { :type => "MIT", :file => "LICENSE" }
+s.author             = { "Bogdan Todea-Cristea" => "bogdan.todea@3pillarglobal.com" }
+s.platform     = :ios, "9.0"
+s.source       = { :git => "https://github.com/3pillarlabs/ios-autocomplete.git", :tag => "#{s.version}" }
+s.source_files  = "Framework/AutoCompletion/AutoCompletion", "Framework/AutoCompletion/AutoCompletion/**/*.{h,m}"
+s.exclude_files = "Classes/Exclude"
```
Comment (david-livadaru-3pillar) [inline AutoCompletion.podspec]: Remove exclude_files, does not seem to be needed.

File: AutoCompletion.podspec [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Pod::Spec.new do |s|
+s.name         = "AutoCompletion"
+s.version      = "1.2.1"
+s.summary      = "Text auto completion framework."
+s.description  = "Library can be used via cocoapods in any project."
+s.homepage     = "https://github.com/3pillarlabs/ios-autocomplete/tree/AutoCompleteFramework"
```
Comment (david-livadaru-3pillar) [inline AutoCompletion.podspec]: `https://github.com/3pillarlabs/ios-autocomplete`should be the home page.

File: AutoCompletion.podspec (hunk 1 above)
Comment (BogdanTC) [reply AutoCompletion.podspec]: Changed.

File: AutoCompletion.podspec (hunk 2 above)
Comment (BogdanTC) [reply AutoCompletion.podspec]: Updated.

**Comment by david-livadaru-3pillar** [comment]
Since we're providing the framework through dependency managers, we should remove the compiled framework since that was causing the in issue #18.

**Review by david-livadaru-3pillar — APPROVED** [review]

**Pull request merged**